test/nightwatch_tests/custom-commands/populateDatabase.js

```diff
@@ -9,25 +9,40 @@ function PopulateDatabase () {
 
 util.inherits(PopulateDatabase, events.EventEmitter);
 
-PopulateDatabase.prototype.command = function (databaseName) {
+PopulateDatabase.prototype.command = function (databaseName, count) {
   var that = this,
       nano = helpers.getNanoInstance(),
       database = nano.use(databaseName),
       i = 0;
 
   async.whilst(
-    function () { return i < 20; },
+    function () { return i < (count ? count : 20); },
     function (cb) {
-        i++;
-        var document_id = 'document_'+ i;
-        database.insert({ number: i }, document_id, cb);
+      i++;
+      var document_id = 'document_'+ i;
+      database.insert({ number: i }, document_id, cb);
     },
     function (err) {
       if (err) {
         console.log('Error in nano populateDatabase Function: ' +
-          document_id + ', in database: ' + databaseName, err.message);
```

Comment:
as `document_id ` is not in this scope it threw an error in case of an error not displaying the error message